app/views/layouts/_error_messages.html.erb

```diff
@@ -0,0 +1,40 @@
+<section class="status-message">
+
+  <% if flash[:success] %>
+    <h3 class="success">
+      <%= flash[:success] %>
+    </h3>
+  <% end %>
+
+  <% if flash[:failure] %>
+    <h3 class="failure">
+      <%= flash[:failure] %>
+    </h3>
+  <% end %>
+
+
+  <% if flash[:failure] == "Could not upvote" %>
+    <ul>
+      <li>
+        user: has already voted for this work
+      </li>
+    </ul>
+  <% end %>
+
+
+  <% if @work %>
+    <% @work.errors.messages.each do |field, messages| %>
+      <% if messages.empty? %>
```

Comment:
You should never have an empty `if` -- if you only need to act in one of these situations, then you should use the inverse condition: `!messages.empty?` in this case